subcommands/targets/list.go

```diff
@@ -52,16 +53,36 @@ func (t byTargetKey) Less(i, j int) bool {
 	return verI < verJ
 }
 
+type column struct {
+	Formatter func(tl *targetListing) string
+}
+
+var Columns = map[string]column{
+	"version":      {func(tl *targetListing) string { return strconv.Itoa(tl.version) }},
+	"tags":         {func(tl *targetListing) string { return strings.Join(tl.tags, ",") }},
+	"apps":         {func(tl *targetListing) string { return strings.Join(tl.apps, ",") }},
+	"hardware-ids": {func(tl *targetListing) string { return strings.Join(tl.hardwareIds, ",") }},
+	"origin":       {func(tl *targetListing) string { return tl.origin }},
+}
+
 func init() {
+	var defCols = []string{"version", "tags", "apps", "origin"}
```

Comment:
We just have 5 columns atm, I think it's fine to show them all by default.
This column selector is neat in any case.

Reply:
it started scrolling off my screen, so i got mad. hardware-id isn't that useful for most customers. they build for a single platform. However, a few factories and things like the public LMP do. So you sometimes need to see, but not usually.